firefox-ios/firefox-ios-tests/Tests/ClientTests/SearchViewControllerTests.swift

```diff
@@ -64,7 +64,24 @@ class SearchViewControllerTest: XCTestCase {
         profile.prefs.setBool(false, forKey: PrefsKeys.FirefoxSuggestShowSponsoredSuggestions )
         profile.prefs.setBool(false, forKey: PrefsKeys.FirefoxSuggestShowNonSponsoredSuggestions)
         await searchViewController.loadFirefoxSuggestions()?.value
-
         XCTAssertEqual(searchViewController.firefoxSuggestions.count, 0)
     }
+
+    func testFirefoxSuggestionsAreFilteredWhenShowSponsoredSuggestionsIsTrue() {
```

Comment:
Nice! 💯

If it's not too cumbersome, could you add a couple for synced tabs, too, please?

Reply:
i had to unset private from `remoteClientTabs` and `filteredRemoteClientTabs`